### PR TITLE
Add some bottom padding to YAML conf dev tools page

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Add some padding, since otherwise the reload table clings to the page bottom.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — DISMISSED** [review]

File: src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts [hunk 1]
```diff
@@ -215,7 +215,7 @@ export class DeveloperYamlConfig extends LitElement {
         }
 
         .content {
-          padding: 28px 20px 0;
+          padding: 28px 20px 16px;
```
Comment (zsarnett) [inline src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts:218]: Should we add margin-bottom instead?

```
margin-bottom: 24px;
margin-bottom: max(24px, env(safe-area-inset-bottom));
```

Comment (spacegaier) [reply src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts:218]: Doesn't really make a difference, does it? So I am fine either way.

Comment (zsarnett) [reply src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts:218]:
```suggestion
          padding: 28px 20px 0;
          margin-bottom: 24px;
          margin-bottom: max(24px, env(safe-area-inset-bottom));
```

**Comment by spacegaier** [comment]
Discussed with Zack: The `margin` approach does not work (even when directly setting it for the `ha-card`). We decided to go back to the `padding` fix.

**Review by zsarnett — APPROVED** [review]

**Pull request merged**